src/server/scripts/EasternKingdoms/zone_loch_modan.cpp

```diff
@@ -4,91 +4,13 @@
  * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
```

Comment:
I guess this file actually could be deleted, not useful to have an empty file lying around :D

@helias ?

Reply:
I'll need help deleting the file because when I CMake he says the file is missing, so that i'm unsure how to procede with deletion

Reply:
I'll look into it

Reply:
works for me, I just deleted it and re-ran CMake
https://gist.github.com/Kitzunu/ad7dc64f3c6844d76c79e8c404029410

Reply:
Nvm I see now... the error comes when you compile and VS runs Cmake....

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/f4953cc13b3c986b703e9ebc95fd82aaf385ef24/src/server/scripts/EasternKingdoms/CMakeLists.txt#L129

Remove this line @gengarshadowball and it will work

Reply:
nice catch 👍 i'm a noob with cmake